src/guides/v2.3/config-guide/revert-split-database.md

```diff
@@ -0,0 +1,69 @@
+---
+group: configuration-guide
+title: Configure a single database from a split database
+ee_only: true
+functional_areas:
+  - Configuration
+  - System
+  - Setup
+---
+
+The [split database]({{ page.baseurl }}/config-guide/multi-master/multi-master.html) feature was deprecated in version 2.4.2 of {{ site.data.var.ee }} and {{ site.data.var.ce }}. Follow these instructions to revert from a split database to a single database implementation.
```

Comment:
Your headers say this is an EE feature. Fix the discrepancy, and if it's available on EE only, add the `edition: ee-only` header to the TOC.

Reply:
The correct metadata tag is `ee_only: true` and already exists on line 4.